tests/LineScanCameraTests.cpp

```diff
@@ -238,7 +238,7 @@ TEST_F(OrbitalLineScanSensorModel, InversionHeight) {
 TEST_F(OrbitalLineScanSensorModel, InversionReallyHigh) {
   for (double line = 0.5; line < 16; line++) {
     csm::ImageCoord imagePt(line, 8);
-    csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 49000.0);
```

Comment:
This was too close to the sensor, it caused the ground to image to diverge because of how close it was.